scalability_and_performance/optimizing-cpu-usage.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="optimizing-cpu-usage"]
+= Optimizing CPU usage with mount namespace encapsulation
+include::_attributes/common-attributes.adoc[]
+:context: optimizing-cpu-usage
+
+toc::[]
+
+You can optimize CPU usage in {product-title} clusters by using mount namespace encapsulation to provide a private namespace for kubelet and CRI-O processes. This reduces the cluster CPU resources used by systemd with no difference in functionality.
+
+:FeatureName: Mount namespace encapsulation 
+include::snippets/technology-preview.adoc[]
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview].
```

Comment:
Noticed while scanning the preview for merge: This statement is also in the TP snippet, so probably want to remove this line. Otherwise ready for merge - you're welcome to message me directly rather than relabeling for merge review when that's done :)

Reply:
Done.